ui/aura/window_tree_host_platform.cc

```diff
@@ -42,8 +42,8 @@ WindowTreeHostPlatform::WindowTreeHostPlatform(const gfx::Rect& bounds)
   bounds_ = bounds;
   CreateCompositor();
 #if defined(USE_OZONE)
-  platform_window_ =
-      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, bounds);
+  platform_window_ = ui::OzonePlatform::GetInstance()->CreatePlatformWindow(
```

Comment:
in order to avoid changing all CreatePlatformWindow calls, maybe add a method to ui/platform_window/platform_window_delegate.h like get_parent?